src/Ganeti/HTools/CLI.hs

```diff
@@ -259,6 +261,7 @@ defaultOptions  = Options
   , optNodeSim     = []
   , optNodeTags    = Nothing
   , optSkipNonRedundant = False
+  , optStaticKvmNodeMemory = 4096
```

Comment:
I don't know if using this as a default is a good idea, though. If someone is using KVM with machines with a low amount of memory, this change might break their use case. Do we have an estimate of how much memory is needed in order to run KVM? If that number is somewhat close to 4GB, I think I'd be fine with this.